docs/rst_include/warning-ttl-twcs.rst

```diff
@@ -6,4 +6,6 @@
    * Tombstone compaction can be enabled to remove data from partially expired SSTables, but this creates additional WA (write amplification).
 
 .. caution::
-   Avoid overwriting data and deleting data explicitly at all costs, as this can potentially block an expired SSTable from being purged, due to the checks that are performed to avoid data resurrection.
\ No newline at end of file
+   * Avoid overwriting data and deleting data explicitly at all costs, as this can potentially block an expired SSTable from being purged, due to the checks that are performed to avoid data resurrection.
+   * If a table is guaranteed to have no overwrites or deletions, user can use the :ref:`immediate tombstone GC mode <ddl-tombstones-gc>` for optimal GC efficiency.
+   * If a table has either overwrites or deletions, user must use the :ref:`repair tombstone GC mode <ddl-tombstones-gc>`. It reduces GC efficiency since expired data will be removed only after repair operation completes.
```

Comment:
s/must/should/
The user can still use the timeout mode as before and make sure repair runs within the gc_grace_seconds period.
Also note that even with overwrites (without explicit deletions), if all writes use the same ttl, mis-purging expired data would resurrect older data that would also be expired by this time, so practically there is no risk (given the ttl is the same for all writes and overwrites).

Reply:
Also s/It reduces/It may reduce/, since if repair frequency is high enough, e.g. less than half of the default ttl, then data would be garbage collectible as soon as it expires since repair would have already run successfully once (or more) within the data life time.

Reply:
We are not pushing for users using tombstone-gc=repair too hard just yet, see https://github.com/scylladb/scylladb/issues/24043.